### ISSUE TITLE
MO2 Doesn't Redirect xEdit's Plugin List Reference

### ISSUE THREAD

**Issue by NoDice1** [ModOrganizer2/modorganizer#2150]
## The problem:
Briefly describe the issue you are experiencing. Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to our [Discord server](https://discord.gg/ewUVAqyrQX).

I've been trying to use xEdit for FO4, running through MO2's executable list and when running FO4Edit, it only lists the plugins from the vanilla game plus dlc. From what I've read about similar issues, MO2 should be redirecting the Plugin.txt file found here, (C:\Users\Username\AppData\Local\Fallout4\Plugin.txt) to its own plugin list containing the plugins from the virtual folder. However xEdit still draws from the file path above. Again, after reading other similar issues, the problem seems to be MO2 related. It looks like another user reported a similar issue with SSE: https://github.com/ModOrganizer2/modorganizer/issues/1797
## Environment:

* Mod Organizer Version that exhibits the issue: v2.4.4
* Last Mod Organizer Version that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Mod Organizer: Windows 11

## Details:
If necessary, describe the problem you have been experiencing with more details and pictures.

If it's relevant, I have FO4 installed on my C drive, not in the program files.
![Reference1](https://github.com/user-attachments/assets/050b077a-c29e-437c-8fe4-dfba42ed8283)
![Reference2](https://github.com/user-attachments/assets/6b630051-dee4-4689-887e-e2462ab00009)

## Link to Mod Organizer logs:
Right click on the log window at the bottom of MO2, set log level to debug, restart Mo2 and reproduce the problem.
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder".

### USVFS:
Create a [GIST](https://gist.github.com) which is a paste of your latest USVFS log, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

https://gist.github.com/NoDice1/67e4ccccb9fa21ab8ee1b24fc67a07e3#file-usvfs-2024-10-17_02-07-05-log

### MO Interface:
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

https://gist.github.com/NoDice1/dba4ddfac0defc2324d2bd3a5b45707d

**Comment by Silarn** [comment]
You do have mods installed and enabled, right? Are you able to see any of them in game or in the file explorer (via the executables)?

xEdit doesn't even really need the plugins.txt that would merely indicate what was enabled. It would merely need to see the plug-in files in the directory.

**Comment by NoDice1** [comment]
> You do have mods installed and enabled, right? Are you able to see any of them in game or in the file explorer (via the executables)?
> 
> xEdit doesn't even really need the plugins.txt, that would merely indicate what was enabled. It would merely need to see the plug-in files in the directory.

Yes, mods are installed, enabled and I can see them in game, though all the plugins, textures, etc. are not in FO4's root folder or the file address in my post but rather in MO2's virtual folder.
